### ISSUE TITLE
How transfer the instanceNorm2d to ncnn MVN

### ISSUE THREAD

**Issue by lzx1413** [starimeL/PytorchConverter#1]
It seems that this convert tool will only convert the base operator of pytorch to  other framework, for example, the instanceNorm is composed of a view and batchnorm, so this tool will transfer the view and batchnorm rather than the instanceNorm as a single layer and convert it to the MVN layer in ncnn. So any solutions to solve this problem? thanks a lot.

**Comment by starimeL** [comment]
May I know will it cause a speed difference between splitting InstanceNorm into two and using it as a direct MVN in ncnn? How large is it

**Comment by lzx1413** [comment]
the question is that the process of mvn is different from batchnorm in ncnn, so the result is not the same. Why this converter can not just convert the structure at a appropriate depth level of the model framework, just like the print works in pytorch

**Comment by starimeL** [comment]
OK, I'll later ask the ncnn author and check how to make an exact transfer between these two.

**Comment by jainshobhit** [comment]
Were you able to make an exact transfer for the InstanceNorm layer?
